Mathlib/Algebra/GradedMonoid.lean

```diff
@@ -283,6 +283,16 @@ instance GradeZero.one : One (A 0) :=
   ⟨GOne.one⟩
 #align graded_monoid.grade_zero.has_one GradedMonoid.GradeZero.one
 
+variable [GradedMonoid.GOne A]
+
+@[simp]
+theorem one_fst : (1 : GradedMonoid A).fst = 0 :=
+  rfl
+
+@[simp]
+theorem one_snd : (1 : GradedMonoid A).snd = 1 :=
+  rfl
```

Comment:
I've split these results to #9227, which catches a handful of similar missing statements.